pages/sections/landing.tsx

```diff
@@ -61,7 +70,7 @@ export default function Landing() {
                 </motion.button>
               </Link>
             )}
-          {!session && (
+          {/* {!session && (
```

Comment:
vercel is also complaining about the `toggleSigninModal` function on line 21.

I think u can just comment out the `toggleSigninModal` function (lines 21-23) to make the vercel deployment pass